python intro/calculator/calc.py

```diff
@@ -6,11 +6,11 @@ def mainFunc ():
     var2 = input("Input Second Number: ")
     calcType = input("What do you want to calculate for? ")
 
-    if calcType != "sum" and calcType != "product" and calcType != "difference" and calcType != "quotient":
+    if calcType not in ["sum", "product", "difference", "quotient"]:
```

Comment:
`a not in b` is saying, if a is not an element of b

Reply:
Makes sense, thanks.